src/station_gui.cpp

```diff
@@ -1517,15 +1517,15 @@ struct StationViewWindow : public Window {
 	 * even if we actually don't know the destination of a certain packet from just looking at it.
 	 * @param i Cargo to recalculate the cache for.
 	 */
-	void RecalcDestinations(CargoType i)
```

Comment:
Here you start using `cargo_type` for the rest of the file.

Reply:
Using `cargo` for `CargoDataEntry` is weird. Other places use `entry`.

Reply:
Yeah, I was going to do this and then got sidetracked. I'll fix it tonight :-)